src/gpl/src/replace.tcl

```diff
@@ -384,4 +388,18 @@ proc get_global_placement_uniform_density { args } {
   }
   return $uniform_density
 }
+
+proc get_mbff_results { args } {
```

Comment:
Why is the command get_mbff_results but it invokes replace_run_mbff_cmd.  The user command should be something like cluster_flops and should be in the global namespace not the gpl namespace.